charts/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 0.12.0
+version: 0.12.1-alpha.1
```

Comment:
@jpweber @sjmiller609 what should we do here?

@jpweber for some background, this chart is bundled into the published umbrella chart. We aren't actually publishing this chart directly.

Bumping these versions as part of commits always feels off to me. :thinking:

Reply:
got it. I was following the directions in the PR template, but I had to ask Steven about it so I might have gotten it wrong still. Just wasn't clear what I should be bumping.

Reply:
it honestly does not matter very much until the "approve release" button is pressed.

Reply:
So when would this get set to something normal? I feel like we should just pull this out for now, what do ya'll think?

Reply:
```suggestion
version: 0.12.0
```